src/core/api/index.js

```diff
@@ -156,6 +157,9 @@ export function callApi({
       options.headers.authorization = `Bearer ${apiState.token}`;
     }
   }
+  if (_config.get('server') && apiState.regionCode) {
```

Comment:
Why is `_config.get('server')` needed exactly?

Reply:
It's not needed, but we only need to do this for calls from the server, so I figured it made sense to add it.

Reply:
so client api requests don't need to pass that header?

Reply:
I'm pretty sure client requests don't need them, as the nginx server receiving the requests will add it, but we can check with @diox.

We can certainly add them all the time, if you think that would make the code cleaner. In fact, my first iteration did just that, and then I realized that we only need to add them on the server.

Reply:
I'd prefer to keep a common behavior yeah